Description: Protect header wrapping against reflowing
 This changes the line wrapping of headers to ensure that the first line 
 of rewrapped headers do not just consist of the header name but instead 
 contains the first bit of the value.
 .
 This ensures that programs that rewrap or reflow headers (such as 
 mailman) do not accidentally result in headers that are longer than 
 998 characters.
Author: Arthur de Jong <adejong@debian.org>
Bug-Debian: https://bugs.debian.org/789616

--- a/camel/camel-mime-utils.c
+++ b/camel/camel-mime-utils.c
@@ -5236,12 +5236,9 @@ camel_header_fold (const gchar *in,
 				g_string_truncate (out, out->len - 1);
 				g_string_append_c (out, '\n');
 				g_string_append_c (out, spc);
-			} else {
-				g_string_append (out, "\n\t");
+				outlen = 1;
 			}
 
-			outlen = 1;
-
 			/* check for very long words, just cut them up */
 			while (outlen + len > CAMEL_FOLD_MAX_SIZE) {
 				tmplen = CAMEL_FOLD_MAX_SIZE - outlen;
